### PR TITLE
Add Extract action

### PR DESCRIPTION
**Pull request by patrick-tolosa**
#### What does this PR solve?
Align transformation.extract()

#### Final checklist
- [X] JSdoc - Add proper docstrings based on our PHP implementation
- [X] Implementation is aligned to Spec.
- [X] Tests - Add proper tests to the added code

### CODE REVIEW

File: src/actions/extract/Extract.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+class Extract extends Action {
+  private qualifierValue = new QualifierValue();
+
+  constructor() {
+    super();
+    this.qualifierValue.delimiter = ';';
+  }
+
+  /**
+   * @description Delivers an image containing only specified layers of a Photoshop image.
```
Comment (strausr) [inline src/actions/extract/Extract.ts:19]: Maybe change docstring to Extracts? instead of delivers

File: src/actions/extract/Extract.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+   * <b>Learn more:</b> {@link https://cloudinary.com/documentation/paged_and_layered_media#deliver_selected_layers_of_a_psd_image | Deliver selected layers of a PSD image}
```
Comment (strausr) [inline src/actions/extract/Extract.ts:20]: Maybe change docstring to Extracts? instead of delivers

File: src/transformation/Transformation.ts [hunk 3]
```diff
@@ -254,6 +255,15 @@ class Transformation {
     return this.addAction(action);
   }
 
+  /**
+   * @doc
+   * @description Adds a page or frame from a document
```
Comment (strausr) [inline src/transformation/Transformation.ts:260]: here also would change to extracts just so naming is more clear to whoever is using

**Review by strausr — APPROVED** [review]

File: src/transformation/Transformation.ts (hunk 3 above)
Comment (patrick-tolosa) [reply src/transformation/Transformation.ts:269]: Changed

File: src/actions/extract/Extract.ts (hunk 2 above)
Comment (patrick-tolosa) [reply src/actions/extract/Extract.ts:20]: Changed

**Pull request merged**